Gemfile

```diff
@@ -2,4 +2,4 @@ source 'https://rubygems.org'
 
 gem 'jekyll'
 gem 'jekyll-redirect-from'
-gem 'rouge'
+gem 'rouge', :git => 'https://github.com/kylef/rouge', :branch => 'kylef/apiblueprint-lexer'
```

Comment:
I did not notice about your PR in main repository. Sorry. Let's wait a week and see what is going to happen.